numpy/f2py/crackfortran.py

```diff
@@ -3335,14 +3353,23 @@ def vars2fortran(block, vars, args, tab='', as_interface=False):
 ######
 
 
+# We expose post_processing_hooks as global variable so that
+# user-libraries could register their own hooks to f2py.
+post_processing_hooks = []
```

Comment:
Could this be mentioned in the main PR text as well?

Reply:
Yes, makes sense. Done.